src/main/resources/templates/dashboard.html

```diff
@@ -92,7 +92,7 @@ <h2 style="text-align: center; display: block">Your Activity</h2>
                                     <tbody>
                                     <tr th:each="notification : ${notifications}">
                                         <td th:text="${#temporals.format(notification.getCreatedAt(), 'dd.MM.yyyy, HH:mm')}"></td>
-                                        <td th:text="${notification.getMessage()}"></td>
+                                        <td th:utext="${notification.getMessage()}"></td>
```

Comment:
To nie bardzo zaciekawiło. Co daje th:utext zamiast th:text?

Reply:
W tym miejscu przychodzący string zawiera tagi htmlowe. Utext pomaga zbudować z tego elementy dom. Ze zwykłym text byśmy wyświetlili to wszystko jako tekst